src/runtime/actor/mod.rs

```diff
@@ -40,10 +40,36 @@ pub trait Actor: Send + Sync + Sized {
 
     /// Run the actor event loop
     async fn run(&mut self, cx: &mut ActorContext<Self>, state: &mut Self::State) -> Result<(), Self::Error> {
+        #[cfg(feature = "metrics")]
+        let histogram = {
+            let histogram = prometheus_client::metrics::histogram::Histogram::new(
```

Comment:
I opened https://github.com/iotaledger/bee/pull/1410 to "fix" this :)

Reply:
@Alexandcoats I think you can update `bee-metrics` in your branch and use `histogram` now.